src/main/deploy/pathplanner/generatedJSON/MoveOneMeters.wpilib.json

```diff
@@ -0,0 +1 @@
locity":0.0,"holonomicAngularVelocity":0.0},{"time":11.630285661427033,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.18731465570693773,"y":0.0}},"velocity":0.2722196983369495,"acceleration":-0.24999999999999425,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":11.673206461945586,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.17586104272704106,"y":0.0}},"velocity":0.26148949820731127,"acceleration":-0.24999999999999742,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":11.720894188484158,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.16367546795162458,"y":0.0}},"velocity":0.24956756657266826,"acceleration":-0.2500000000000035,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":11.77409600003516,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.15075182539962206,"y":0.0}},"velocity":0.23626711368491796,"acceleration":-0.24999999999999634,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":11.833832961114524,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.13708400908996698,"y":0.0}},"velocity":0.2213328734150768,"acceleration":-0.25000000000000044,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":11.901566090490876,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.12266591304159302,"y":0.0}},"velocity":0.20439959107098876,"acceleration":-0.25000000000000083,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":11.979521817324471,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.10749143127343364,"y":0.0}},"velocity":0.18491065936258985,"acceleration":-0.25000000000000105,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":12.071418020294148,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.09155445780442295,"y":0.0}},"velocity":0.16193660862017087,"acceleration":-0.24999999999999759,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":12.184439409377656,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.07484888665349443,"y":0.0}},"velocity":0.1336812613492936,"acceleration":-0.2500000000000017,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":12.336948971488624,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.05736861183958175,"y":0.0}},"velocity":0.09555387082155174,"acceleration":-0.24999999999999936,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0},{"time":12.71916445477483,"pose":{"rotation":{"radians":0.0},"translation":{"x":-0.03910752738161816,"y":0.0}},"velocity":0,"acceleration":-0.25000000000000056,"curvature":0,"holonomicRotation":0.0,"angularVelocity":0.0,"holonomicAngularVelocity":0.0}]
```

Comment:
Rename from MoveOneMeters to MoveOneMeter?

Reply:
Done